quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/QuarkusPropertiesDistTest.java

```diff
@@ -123,7 +123,7 @@ void testUnknownQuarkusBuildTimePropertyApplied(LaunchResult result) {
     }
 
     @Test
-    @Launch({ "start", "--http-enabled=true", "--hostname-strict=false", "--config-keystore=keystore" })
+    @Launch({ "start", "--http-enabled=true", "--hostname-strict=false", "--config-keystore=../../../../src/test/resources/keystore" })
```

Comment:
Just to be sure, this change was necessary because the exception about invalid keystore path would take a precedence?

I haven't gone through the recent smallrye-keystore changes, but this might indicate that there are some improvements when it comes to exception handling, so [our custom property validation](https://github.com/keycloak/keycloak/blob/0c44b28b631aea25c2e09043a9b047cf1147c0a5/quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/ConfigKeystorePropertyMappers.java#L41) might be redundant to some extent.

Reply:
Ok then, thanks for clarification.